frontend/src/components/CreateInstanceForm.vue

```diff
@@ -113,6 +113,17 @@
             @input="handleInstancePortInput"
           />
         </div>
+
+        <template v-if="state.instance.engine == 'MONGODB' && isDev()">
```

Comment:
Seems we can remove the `isDev()` here as we already take the check for the engine list